.github/workflows/ci.yml

```diff
@@ -101,7 +101,7 @@ jobs:
           export OS_TYPE=${{ matrix.os_type }} 
           curl -sSfL https://raw.githubusercontent.com/${{ github.repository }}/${{ github.sha }}/download.sh -O
           sh ./download.sh
-            test "$(ls -A ./otel-dotnet-auto)"
+            test "$(ls -A "$HOME/.otel-dotnet-auto")"
```

Comment:
Want to understand this better, do we need `/.` or `/` would be sufficient here?

Reply:
dotnet-install.sh does the same. It makes the dir "hidden". `ls` will not list it, but `ls -a` will list it.